elements/rh-secondary-nav/demo/analytics.html

```diff
@@ -0,0 +1,209 @@
+<link rel="stylesheet" href="../demo.css">
+<link rel="stylesheet" href="../../rh-secondary-nav-lightdom.css">
+<script type="module" src="../rh-secondary-nav.js"></script>
+
+<div class="mock-nav"></div>
+<rh-secondary-nav role="navigation" data-analytics-region="Ansible secondary nav">
+  <a href="#" slot="logo" id="logo-id">
+    Red Hat Ansible Automation Platform
+  </a>
+  <ul slot="nav" aria-labelledby="logo-id">
+    <li>
+      <rh-secondary-nav-dropdown>
+        <a href="#" data-analytics-level="1" data-analytics-text="Explore" slot="link">Explore</a>
```

Comment:
Are `data-analytics-level="1"` attributes still valuable for us to capture?

Reply:
No we should remove all those.  `data-analytics-level`